### DISCUSSION TITLE
Append airport code in DLC-GS addresses

### DISCUSSION THREAD

**Discussion by olyair11** [szpajder/dumpvdl2#34] [General]
Hello,
Tried to enrich DLC addresses, appending the airport code of pointing GS address.
The code is taken from the imported GS database file.

https://github.com/szpajder/dumpvdl2/compare/unstable...olyair11:dumpvdl2:unstable

Please check if applicable
Thanks

**Comment by szpajder** [comment]
TBH, I'm not a fan of it. Neither the feature, nor the way it's implemented.
